### DISCUSSION TITLE
"Explore" button disappeared from SQL Lab

### DISCUSSION THREAD

**Discussion by timurkh** [apache/superset#17142] [Q&A / Help]
Not sure what caused this, but now I do not have "Explore" button in the SQL Lab, which I used to create virtual datasets, only "download to CSV" and "copy to clipboard". I have latest version of the superset launched using docker-compose, no errors or warnings in the log, and I use postgres database to store data. Week ago this button was in place, perhaps the only significant change since those times - I deleted all example charts/datasets and database, which superset created on the first start.

Any ideas what I could check are very appreciated.

**Comment by anhnttimo** [comment]
Does your queries run with record returned?

**Reply by timurkh** [reply]
yes, of course
actually after upgrade button returned back, not sure what was that

**Reply by eschutho** [reply]
@timurkh is this all working now?

**Reply by timurkh** [reply]
@eschutho - yes, now everything works, thanks!
